roxie/roxiemem/roxiemem.hpp

```diff
@@ -393,7 +393,7 @@ class OwnedRoxieString
     inline const char * getLink() const { LinkRoxieRow(ptr); return ptr; }
     inline const char * set(const char * _ptr) { const char * temp = ptr; if (_ptr) LinkRoxieRow(_ptr); ptr = _ptr; ReleaseRoxieRow(temp); return ptr; }
     inline const char * setown(const char * _ptr) { const char * temp = ptr; ptr = _ptr; ReleaseRoxieRow(temp); return ptr; }
-
+    inline void clear() { ReleaseRoxieRow(ptr); ptr = NULL; }
```

Comment:
I recoded as

    inline void clear() { const char * temp = ptr; ptr = NULL; ReleaseRoxieRow(temp);  }

for consistency with other implementations of clear() above